awscli/customizations/wizard/ui/section.py

```diff
@@ -86,3 +101,55 @@ def _create_prompts_from_section_definition(self):
 
     def __pt_container__(self):
         return self.container
+
+
+class WizardDetailsPanel:
```

Comment:
For now, it might make sense to move the detail panels to the `layout` module as it is created once for the layout and is then shared across all of the sections/prompts. I'm also fine if we created a `details.py` module if we have more details related logic we want to consolidate. Mainly it is a little weird that is being grouped with the sections right now.